content/en/docs/contribute/localization.md

```diff
@@ -13,53 +13,66 @@ card:
 
-Then, clone the website repo and `cd` into it:
+Then, clone your fork and `cd` into it:
 
 ```shell
-git clone https://github.com/kubernetes/website
+git clone https://github.com/<username>/website
 cd website
 ```
 
-{{< note >}}
-Contributors to `k/website` must [create a fork](/docs/contribute/start/#improve-existing-content) from which to open pull requests. For localizations, we ask additionally that:
+### Open a pull request
 
-1. Team approvers open development branches directly from https://github.com/kubernetes/website.
-2. Localization contributors work from forks, with branches based on the current development branch.
+Next, [open a pull request](https://kubernetes.io/docs/contribute/start/#submit-a-pull-request) (PR) to add a localization to the `kubernetes/website` repository. 
 
-This is because localization projects are collaborative efforts on long-running branches, similar to the development branches for the Kubernetes release cycle. For information about localization pull requests, see ["branching strategy"](#branching-strategy).
-{{< /note >}}
+The PR must include all of the [minimum required content](#minimum-required-content) before it can be approved.
 
-### Find your two-letter language code
+For an example of adding a new localization, see the PR to enable [docs in French](https://github.com/kubernetes/website/pull/8636).
+    
+### Join the Kubernetes GitHub organization
+
+Once you've opened a localization PR, you can become members of the Kubernetes GitHub organization. Each person on the team needs to create their own [issue requesting membership](https://github.com/kubernetes/org/issues/new/choose) in the `kubernetes/org` repository.
+
+### Add your localization team in GitHub
+
+Next, add your Kubernetes localization team to [`sig-docs/teams.yaml`](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml).
+
+For an example of adding a localization team, see the PR to add the [Spanish localization team](https://github.com/kubernetes/org/pull/685). 
+    
+### Configure the workflow
+
+Next, add a GitHub label for your localization in the `kubernetes/test-infra` repository. A label lets you filter issues and pull requests for your specific language.
+
+For an example of adding a label, see the PR for adding the [Italian language label](https://github.com/kubernetes/test-infra/pull/11316).
+
+### Find community
+
+Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/). Other localization teams are happy to help you get started and answer any questions you have.
 
-Consult the [ISO 639-1 standard](https://www.loc.gov/standards/iso639-2/php/code_list.php) for your localization's two-letter country code. For example, the two-letter code for German is `de`.
+You can also create a Slack channel for your localization in the in `kubernetes/community` repository.
```

Comment:
Maybe good to give an example as well: https://github.com/kubernetes/community/pull/3605 ;) you were the first non-slack admin using Tempelis, and it seems to be working, so we can happily include it as an example IMO 😃

Reply:
@rlenferink It's already there, on line 73. 😄